### PR TITLE
Updations/Corrections in cli options "-h" and "-t".

### PR DESCRIPTION
**Pull request by jkottiku**
Improvement documentation, corrections and indentation for "--help" (-h) and  "--listTests" (-t) options for **rvs** command line.

### CODE REVIEW

File: gst.so/src/rvs_module.cpp [hunk 1]
```diff
@@ -51,51 +51,51 @@ extern "C" int rvs_module_has_interface(int iid) {
 }
 
 extern "C" const char* rvs_module_get_description(void) {
-    return "ROCm Validation Suite GST module";
+  return "The GPU Stress Test runs a Graphics Stress test or SGEMM/DGEMM/HGEMM (Single/Double/Half-precision General Matrix Multiplication) \n\tworkload on one, some or all GPUs.";
```
Comment (manoj-freyr) [inline gst.so/src/rvs_module.cpp:54]: I dont think "graphics stress test" is right. sufficient to say sgemm etc right?

File: src/rvshsa.cpp [hunk 2]
```diff
@@ -342,7 +341,7 @@ void rvs::hsa::InitAgents() {
 
   std::sort(size_list.begin(), size_list.end());
 
-  PrintTopology();
+//  PrintTopology();
```
[the remark this answers is not in the document]
Comment (jkottiku) [reply src/rvshsa.cpp:344]: Topology gets printed while running "rvs -t" option (to list test modules) which is not expected.

File: gst.so/src/rvs_module.cpp (hunk 1 above)
Comment (jkottiku) [reply gst.so/src/rvs_module.cpp:54]: Thats true. The description was taken from FEATURES.md. Will be correcting the description in both the places.

**Review by manoj-freyr — APPROVED** [review]
looks good, see if you can change the " Graphics Stress test " part in both features and desciption message.

**Review by manoj-freyr — APPROVED** [review]
looks good

**Pull request merged**